R/utils.R

```diff
@@ -24,12 +24,6 @@ dev_suppress <- function(x) {
   force(x)
 }
 
-format_expression <- function(code) {
-  code <- lang2calls(code)
-  paste(code, collapse = "\n")
-}
-
```

Comment:
Did you consider having `lang2calls` return the character string itself?

It seems that it is only used this way in the package _(other than in tests)_

ps. and rename it to `lang2str` or `lang2character`?